### ISSUE TITLE
Keys Framework comes to visible after every pod install

### ISSUE THREAD

**Issue by madnik** [orta/cocoapods-keys#130]
Even after I manually hide the Keys scheme. Keys framework appear under scheme list after every pod install. 
<img width="375" alt="screen shot 2016-03-09 at 4 36 12 pm" src="https://cloud.githubusercontent.com/assets/4954307/13629646/732c302a-e615-11e5-9657-a40f55b76293.png">

**Comment by madnik** [comment]
Having 
`:target =>` 
set in the pod file solves this issue. Please close if no other comments to be made.

**Comment by orta** [comment]
Yep, this is an annoying side-effect of Xcode, I put some more effort into trying to remove this lately, but have never found a good way to fix it.

**Comment by pavankatariaog** [comment]
Hi there, great library. What's the update on this? Is this still the case?
